src/Text-Core/RunArray.extension.st

```diff
@@ -0,0 +1,42 @@
+Extension { #name : #RunArray }
+
+{ #category : #'*Text-Core' }
+RunArray class >> scanFrom: strm [
```

Comment:
is this one really used? Since `writeScanOn` do not exist anymore?

Reply:
It has one non test sender that is PositionableStream>>#nextChunkText

Reply:
Maybe update the reference to `writeScanOn` in the comment?

Reply:
I can also move it to the only package using it instead of Text-Core

Reply:
In fact the sender is dead. We can kill all the scanning methods :)
But I'll do that in another PR if it's alright

Reply:
+1

You're the boss, I'm only a commenter :)